src/flint/fmpz.jl

```diff
@@ -1139,19 +1139,22 @@ end
 issquare(x::fmpz) = Bool(ccall((:fmpz_is_square, :libflint), Cint,
                                (Ref{fmpz},), x))
 
+isprime(x::UInt) = Bool(ccall((:n_is_prime, :libflint), Cint, (UInt,), x))
+
 @doc Markdown.doc"""
-    is_prime(x::UInt)
+    isprime(x::fmpz)
 > Return `true` if $x$ is a prime number, otherwise return `false`.
 """
-is_prime(x::UInt) = Bool(ccall((:n_is_prime, :libflint), Cint, (UInt,), x))
+isprime(x::fmpz) = Bool(ccall((:fmpz_is_prime, :libflint), Cint,
```

Comment:
The docs for this function says:
 
> The function assumes that n is likely prime, i.e. it is not very efficient if n is composite.
> A strong probable prime test should be run first to ensure that n is probably prime.

Shall we call `fmpz_is_probabprime` first?
Also, the docs aknowledge a possibility of failure, when it returns `-1`. Then the conversion to `Bool` will fail, should we handle gracefully this case with a clear error message?

Reply:
@wbhart can hopefully answer this.

Reply:
The documentation appears to be correct. A probable prime test is far cheaper than an isprime test, so no problems there.

Reply:
ok, will do